### PR TITLE
NCC-112 Mixed Paper tests are showing the max amount of sample questi…

### PR DESCRIPTION
**Pull request by narekxxxxx**
Related to task: https://oat-sa.atlassian.net/browse/NCC-112

Description: Mixed Paper tests are showing the max amount of sample questions, not the set amount

Requires: https://github.com/oat-sa/extension-tao-booklet/pull/93

How to test: Login as nccer admin, choose child organization after this enter Paper tests and press button generate Pdf . Check amount of sample questions in pdf file

### CODE REVIEW

**Review by ricproenca-tao — CHANGES_REQUESTED** [review]
Can you please update the Updater script in `/scripts/update/Updater.php`

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -45,5 +45,6 @@ public function update($initialVersion)
         }
 
         $this->skip('1.4.0', '1.8.3');
+        $this->skip('1.8.3', '1.9.0');
```
Comment (alexeyvy) [inline scripts/update/Updater.php]: no need for 1 more skip(), please use the one above 

        $this->skip('1.4.0', '1.9.0');

**Review by alexeyvy — APPROVED** [review]

**Review by tikhanovichA — APPROVED** [review]
Code looks good

**Review by AndreyNegrov — APPROVED** [review]
Good job!
[ ] New code is covered by tests (if applicable)
[ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
[x] New code is respecting code style rules
[x] New code is respecting best practices
[x] New code is not subject to concurrency issues (if applicable)
[x] The feature is working correctly on my local machine (if applicable)
[x] Acceptance criteria are respected
[x] Pull request title and description are meaningful

**Review by ricproenca-tao — APPROVED** [review]

**Comment by AndreyNegrov** [comment]
This PR was successfully merged to develop branch but for some reason github doesn't see it.
The develop branch contains all the commits from this pr. For this reason I close this commit manually

**Pull request closed without merging**